### ISSUE TITLE
Getting error  "no network connection"

### ISSUE THREAD

**Issue by Crysalist** [mnill/cordova-app-updater#4]
Getting error  "no network connection". Used an example ionic app checked manifest and 

```
<uses-permission android:name="android.permission.INTERNET" />
<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
<uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
```

Is present. Checked the server and the config.json file and other files exists on the server.\

please advise.

**Comment by mnill** [comment]
With server url 'http://mnillstone.com/' works? You can change server url manualy in plugins/com.likesmagia.cordova.updater/www/updater.html

---

```
        Updater = new H5AppFS(fileSystem, {}, 'http://mnillstone.com/');
```

---

**Comment by Crysalist** [comment]
Changed it - no luck. Keep on getting the same error. Did clean Install and still no luck.

**Comment by mnill** [comment]
Something has been changed in installation pipeline.

Plugin copied from

---

 project/platfroms/android/platfrom_www

---

 not from

---

 project/plugins/com.likesmagia.cordova.updater/www/

---

For testing you can change $server_url in project/platfroms/android/platfrom_www/updater.html

I will be repaired it asap.

**Comment by Crysalist** [comment]
Step to recreate the issue:

<ol>
<li>ionic start myApp blank</li>
<li>cd myApp</li>
<li>ionic platform add android</li>
<li>cordova plugin add https://github.com/mnill/cordova-app-updater.git --variable SERVER_ADDRESS="http://mnillstone.com/"</li>
<li>ionic build android</li>
</ol>

Then I transfer the apk to my phone. Install it and run it. I see a splashscreen and then popup "No Network Connection" with retry button.

Hope this helps

**Comment by Crysalist** [comment]
Once I changed the files in <b>project/platfroms/android/platfrom_www</b> It worked 100%.

looked at adding some progress updates ... will push it back to you once I am done.

**Comment by mnill** [comment]
Can you please also test my fix for installation?

**Comment by Crysalist** [comment]
Seems not:

> ionic build android
> Running command: "C:\Program Files\nodejs\node.exe" C:\Users\Carlo\myApp\hooks\after_prepare\010_add_platform_class.js C:\Users\Carlo\myApp
> add to body class: platform-android
> Error: App update could not find server_adress

Checked both places 'com.likesmagia.cordova.updater/www' and 'platform_www' still have $SERVER_ADDRESS in them

**Comment by mnill** [comment]
:-( mac or windows?

Edit: windows, i see

**Comment by Crysalist** [comment]
windows

**Comment by mnill** [comment]
okay, i give up. I can't find not ugly way to inject server_url on plugin installation. 
Than i just fixed readme.

**Comment by mnill** [comment]
Resolved in 0.9
